app/validations/user-email.ts

```diff
@@ -0,0 +1,23 @@
+// import { computed } from '@ember/object';
```

Comment:
@baylee-d, I am not sure how this pertains to the PR intent.

Reply:
You're completely right, accidentally added, this sorry about that. Removing.